runtime/doc/lsp.txt

```diff
@@ -2529,6 +2533,27 @@ symbols_to_items({symbols}, {bufnr}, {position_encoding})
 ==============================================================================
 Lua module: vim.lsp.log                                              *lsp-log*
 
+The `vim.lsp.log` module provides logging for the Nvim LSP client.
+
+When debugging language servers, it is helpful to enable extra-verbose logging
+of the LSP client RPC events. Example: >lua
+    vim.lsp.set_log_level 'trace'
+    require('vim.lsp.log').set_format_func(vim.inspect)
+<
+
+Then try to run the language server, and open the log with: >vim
+    :lua vim.cmd('tabnew ' .. vim.lsp.get_log_path())
+<
+
+(Or use `:LspLog` if you have nvim-lspconfig installed.)
```

Comment:
Do we want to mention this?

Reply:
Normally I wouldn't, but I did this partially as a reminder that (1) we need to [upstream](https://github.com/neovim/neovim/issues/28479) these in some fashion , and (2) to update this when they eventually are upstreamd.